GPflowOpt/acquisition.py

```diff
@@ -161,6 +172,10 @@ def __init__(self, model):
     def constraint_indices(self):
         return np.arange(self.data[1].shape[1])
 
+    def feasible_data_index(self):
+        pred = self.evaluate(self.data[0])
+        return pred.ravel() > 0.5
```

Comment:
I actually don't mind some feedback from the GPFlow people on what's the standard way to handle constraints or noise in machine learning.